website/docs/r/cosmosdb_postgresql_cluster.html.markdown

```diff
@@ -43,6 +43,8 @@ The following arguments are supported:
 
 * `coordinator_storage_quota_in_mb` - (Required) The coordinator storage allowed for the Azure Cosmos DB for PostgreSQL Cluster. Possible values are `32768`, `65536`, `131072`, `262144`, `524288`, `1048576`, `2097152`, `4194304`, `8388608` and `16777216`.
 
+-> **NOTE:** For more information, please see the [product documentation](https://learn.microsoft.com/azure/cosmos-db/postgresql/resources-compute)
```

Comment:
This is unclear as to what we're linking too - more information on _what_ exactly?

A little more context would make this clearer, for example:

```suggestion
-> **NOTE:** More information on [the types of compute resources available for CosmosDB can be found in the product documentation](https://learn.microsoft.com/azure/cosmos-db/postgresql/resources-compute)
```

Reply:
Thanks. Updated.